src/blockchain/BitcoinBlockchain.ts

```diff
@@ -67,8 +69,34 @@ export class BitcoinBlockchain {
      * @returns BitcoinFee
      */
     public async GetTxFee(): Promise<BitcoinFee> {
-        var fees = await this._prokeyBtcBlockchain.GetTxFee();
         var fee = <BitcoinFee>{};
+        if (this._prokeyBtcBlockchain._coinName === 'BTC')
+        {
+            try {
+                
+                // get fee from https://bitcoinfees.earn.com/api/v1/fees/list            
+                const client = httpclient.newHttpClient();
+
+                const request = new Request("https://bitcoinfees.earn.com/api/v1/fees/list", {method: 'GET'});
+                
+                var r = await client.execute<any>(request);    
+                r.fees.forEach(element => {
+                    if (element.maxMinutes == 360 && fee.economy == null) {
+                        fee.economy = element.minFee;                        
+                    } else if (element.maxMinutes == 180 && fee.normal == null) {
+                        fee.normal = element.minFee;
+                    } else if (element.maxMinutes == 60 && fee.high == null) {
+                        fee.high = element.minFee;
+                    }
+                });
+
+                return fee;
+            }
+            catch (error) {
```

Comment:
Better to use MyConsole.Log(error) to capture the error message.

Reply:
I think there is no need to log anything here.